RadixWallet/RadixConnect/RadixConnect/Mobile/RadixConnectMobile.swift

```diff
@@ -0,0 +1,42 @@
+import SargonUniFFI
+
+// MARK: - RadixConnectMobile
+/// A very thin wrapper around Sargon.RadixConnectMobile.
+/// It mainly initializes the `live` version Sargon.RadixConnectMobile and
+/// exposes the incomming messages stream.
+struct RadixConnectMobile {
```

Comment:
I guess `P2P.RTCIncomingMessage` does not exist in Sargon? preventing this struct to be implemented in **Swift** Sargon?

What if you make this struct generic?

```swift
public struct RadixConnectMobile<Message> {
  ...
   ... incomingMessagesSubject: AsyncReplaySubject<Message> ...

    func incomingMessages() async -> AnyAsyncSequence<Message> {
		...
	}
}
```

Then it can be placed in **Swift** Sargon, with all the benefits that entails: 
* Easy to write unit tests
* Getting code coverage measurements

Reply:
I don't quite follow your suggestion, why I would make this generic when it is specific to RadixConnectMobile?

`P2P.RTCIncomingMessage` does not exist in Sargon, since it does not yet know about messages from WebRTC or from the Wallet itself; also it is highly specific to iOS.

Reply:
> P2P.RTCIncomingMessage does not exist in Sargon,
that is my point, `P2P.RTCIncomingMessage` does not exist in Sargon. But a generic type `T` can be declared ("exist") in Sargon. Thus this `struct RadixConnectMobile` can exist in Sargon. And having Swift code In Sargon has the above mentioned two advantages:
* Easy to write unit tests
* Getting code coverage measurements

VS the terrible DX of writing unit tests in iOS app repo - without code coverage.

Reply:
I see, we can get to this at some later point when we can unify the channel of incoming message on both platforms.